src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -627,6 +627,13 @@ paths:
     post:
       operationId: ValidatorsUptime
       summary: Get Validators Uptime
+      description: |
+        Returns validators uptime data for time range limited by from_state_version and at_state_version.
+        `at_state_version` defines upper boundary (inclusive) for queried data. i.e `{ at_state_version = {epoch = 10} }`, will return uptime statistics from genesis till 10.
+        `from_state_version` defines lower boundary (inclusive) for queried data. i.e `{ from_state_version = {epoch = 10} }`, will return uptime statistics from epoch 10 till current max ledger tip.
+        it's possible to define both at once i.e `{ at_state_version = {epoch = 10}, from_state_version = {epoch = 25} }` will return uptime statistics for epochs from 10 till 25.
+        you can use one of `state_version`, `epoch`, `epoch` and `round`, or `timestamp` to define from/at state version parameter, but then ongoing epoch will be selected and used for querying data.
+        i.e for request with `{ from_state_version = { state_version = 100 000}, at_state_version = { state_version = 300 000} }` gateway api will check in which epoch transactions with state version 100 000 and 300 000 and then use that as inclusive boundary for request.
```

Comment:
1. I have mixed feeling when it comes to pseudo-json format we used.
2. Same's true when it comes to the formatting of big numbers, maybe it's better to rewrite those samples and use `1000` to `3000` range?